### ISSUE TITLE
Refresh Dataset

### ISSUE THREAD

**Issue by simongiesen** [Tibolte/AgendaCalendarView#15]
Hi,

perhaps I am too silly: How to refresh a dataset? Say, according to your sample app you use  List<CalendarEvent> eventList = new ArrayList<>();

and with 
init( ...) you start. But what's about adding more events to the eventList later on and "informing" the Calendar to repaint or paint the newly added events?

**Comment by LeoJHarris** [comment]
To force the view to draw, call invalidate() on the view

**Comment by Tibolte** [comment]
That's not a silly question, it's a good point we should add a method in order to add more events and refresh the data set. Good catch!

**Comment by simongiesen** [comment]
Great! I will work on it! BTW: When using init() for a refresh after an insert or delete the calendar is not being repaint correctly with it dimensions, no matter if match_parent or fill_parent was used!

**Comment by amannepid** [comment]
I agree with @simongiesen , after some work on agenda data, and trying to refresh the calendar, the calendar takes almost half the size and does not scrolls properly as well. 

It would be a great help if you guys fix this issues asap.

**Comment by ankit2771991** [comment]
yup facing same problem

**Comment by amannepid** [comment]
Did anyone got work around ?

**Comment by simongiesen** [comment]
this needs some basic rework. I stick on it.
- Events for data refresh and data has changed
- Dataset refresh / reload

**Comment by amannepid** [comment]
@simongiesen Did you solved it ? Any suggestions or snippets ?

**Comment by simongiesen** [comment]
ongoing work!

**Comment by amannepid** [comment]
@simongiesen please suggest the workaround you will find. I need it in my current project. :)

**Comment by migzisreallywewwhat** [comment]
any progress on this ?

**Comment by amannepid** [comment]
Hi @Tibolte , I have opened pull request #29, have you checked it out?
